packages/components/src/tools-panel/types.ts

```diff
@@ -16,6 +16,12 @@ export type ToolsPanelProps = {
 	 * The child elements.
 	 */
 	children: ReactNode;
+	/**
+	 * The popover props to configure panel's DropdownMenu.
+	 *
+	 * @default {}
```

Comment:
We can remove this line, since `dropdownMenuProps` is not actually assigned a default value when destructuring the prop (same for the type annotation in `ToolsPanelHeaderProps` and for the README docs)

Reply:
Resolved in https://github.com/WordPress/gutenberg/commit/ea682b935b9bfb9d269133ebacb240c339c15cae.